console/src/main/webapp/manager/app/app.es6

```diff
@@ -17,6 +17,8 @@ class AppController {
         component: 'roles' },
       { path: '/role/:role/:tab',
         component: 'role' },
+      { path: '/delegations/:delegation',
+        component: 'delegations' },
```

Comment:
Will require an update of line
```
RewriteCond %{REQUEST_URI} ^/console/manager/([home|users|org|orgs|roles|logs|analytics].*)$
```
in https://github.com/georchestra/georchestra/blob/master/docs/setup/apache.md

Reply:
Done with e8fe1bd